Gemfile

```diff
@@ -5,12 +5,18 @@ gemspec
 
 group :development do
 	gem 'pry'
-	gem 'pry-coolline'
-	
+  # coolline is a runtime dependency of pry-coolline.
+  # unicode_utils is a runtime dependency of coolline.
+  # unicode_utils requires Ruby '> 1.9' from 1st version.
+  gem 'pry-coolline' if RUBY_VERSION >= '1.9.1'
 	gem 'tty-prompt'
 end
 
 group :test do
 	gem 'simplecov'
-	gem 'coveralls', require: false
```

Comment:
As a reference, this way to describe hash `require: false` is not valid for Ruby 1.8 or Ruby 1.9 (I forget the detail).